sass/components/navbar.sass

```diff
@@ -403,6 +403,10 @@ a.navbar-item,
       padding-top: $navbar-height + ($navbar-padding-vertical * 2)
     &.has-spaced-navbar-fixed-bottom
       padding-bottom: $navbar-height + ($navbar-padding-vertical * 2)
+    &.has-navbar-fixed-top.is-spaced
```

Comment:
What is `is-spaced` supposed to do here?

Reply:
```is-spaced``` extends padding on the desktop. It prevents screen collapse on the desktop when ```navbar``` has ```navbar-fixed-top / bottom``` and ```is-spaced```.